config/wdio/wdio.conf.js

```diff
@@ -22,7 +22,7 @@ const internalPort = process.env.WDIO_INTERNAL_PORT || 8080;
 const ci = process.env.CI;
 
 /* Use to bail fast while running locally. */
-const bail = process.env.WDIO_BAIL || ci;
+const bail = process.env.WDIO_BAIL !== 'FALSE' && (process.env.WDIO_BAIL || ci);
```

Comment:
This change allows me to override the bail flag if the ci flag is set. Previously in my development docker container i'd always have to go and manually edit this config to run all my tests instead of just failing on the first one.

Anything other than FALSE for the WDIO_BAIL env will still set bail to true. But now setting the WDIO_BAIL env to 'FALSE' will not set bail to true too :)